inform7/extensions/basic_inform/Sections/Preamble.w

```diff
@@ -142,6 +142,10 @@ Use dictionary resolution of at least 6 translates as the configuration value
 	DICT_RESOLUTION in BasicInformKit.
 Use no automatic plural synonyms translates as the configuration flag
 	NO_AUTO_PLURAL_NAMES in BasicInformKit.
+Use no status window translates as the configuration flag NO_STATUS_WINDOW
+	in BasicInformKit.
+Use manual line input echoing translates as the configuration flag
+	MANUAL_INPUT_ECHOING in BasicInformKit.
```

Comment:
I added these use options here even though they're Glulx-only. They could be moved to the Glulx and Glk chapter if it's an issue that they're present in Z-Code builds.